### PR TITLE
Fix/link color and missing italic texts

### LINKED ISSUE #1043: Low Saliency of Links
See: https://incidentdatabase.ai/contact

Screen Shot 2022-09-01 at 9 04 16 AM

It is inconsistent on production and some are difficult to detect.

### PR DESCRIPTION
**Pull request by clari182**
- resolves #1043 
- Also fixes missing italic classes

### CODE REVIEW

File: site/gatsby-site/src/components/landing/AboutDatabase .js [hunk 1]
```diff
@@ -10,15 +10,15 @@ export default function AboutDatabase({ className }) {
         <Card.Title as="h2">
           <Trans ns="landing">About the Database</Trans>
         </Card.Title>
-        <Card.Text className="fst-italic">
+        <Card.Text className="tw-italic">
```
Comment (cesarvarela) [inline site/gatsby-site/src/components/landing/AboutDatabase .js:13]: Does this need the prefix?

File: site/gatsby-site/src/components/landing/AboutDatabase .js [hunk 2]
```diff
@@ -10,15 +10,15 @@ export default function AboutDatabase({ className }) {
...
           <Trans i18nKey="aboutLine1" ns="landing">
             The AI Incident Database is dedicated to indexing the collective history of harms or
             near harms realized in the real world by the deployment of artificial intelligence
             systems. Like similar databases in aviation and computer security, the AI Incident
             Database aims to learn from experience so we can prevent or mitigate bad outcomes.
           </Trans>
         </Card.Text>
-        <Card.Text className="fst-italic">
+        <Card.Text className="tw-italic">
```
Comment (cesarvarela) [inline site/gatsby-site/src/components/landing/AboutDatabase .js:21]: ditto

**Comment by clari182** [comment]
Prefix removed @cesarvarela

**Review by pdcp1 — APPROVED** [review]

**Comment by pdcp1** [comment]
@clari182 The links on the login and sign-up page in the PR preview link are not highlighted as expected, please take a look
- https://deploy-preview-1046--staging-aiid.netlify.app/login
- https://deploy-preview-1046--staging-aiid.netlify.app/signup

**Comment by pdcp1** [comment]
@clari182 now it looks good 👍

**Review by pdcp1 — APPROVED** [review]

**Pull request merged**